src/components/download-as/DownloadAs.js

```diff
@@ -10,40 +10,46 @@ class DownloadAs extends PureComponent {
 
     static propTypes = {
         endpoint: PropTypes.string.isRequired,
+        fileTypes: PropTypes.arrayOf(PropTypes.oneOf(['pdf', 'xls', 'csv'])),
+        queryStr: PropTypes.string,
     }
 
-    render() {
-        const api = this.context.d2.Api.getApi()
+    static defaultProps = {
+        fileTypes: ['pdf', 'xls', 'csv'],
+    }
+
+    getHrefs() {
+        const { baseUrl } = this.context.d2.Api.getApi()
+        const { endpoint, fileTypes } = this.props
         const timestamp = Date.now()
+        const queryStr = this.props.queryStr || `t=${timestamp}`
```

Comment:
You explained this during our call, but maybe for posterity it'd be nice to leave a short comment here explaining what's going on with the timestamp, and why we're falling back to that, etc.

Reply:
Is this for cache busting?

Reply:
Yeah that's the only use I can think of, but I'm not sure.

Reply:
I looked into this some more. I do believe this is some sort of cache buster, because if I access the url without the timestamp it still works. Since it was already present I am reluctant to remove it.